core/src/test/scala/unit/kafka/log/LogLoaderTest.scala

```diff
@@ -557,7 +558,7 @@ class LogLoaderTest {
       _topicId = None,
       keepPartitionMetadataFile = true)
 
-    verify(stateManager).removeStraySnapshots(any[Seq[Long]])
+    verify(stateManager).removeStraySnapshots((any[java.util.List[java.lang.Long]]))
```

Comment:
Are the extra parenthesis around `any` needed? We have a few similar examples in this file.